<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewHuntWindow</class>
 <widget class="QWidget" name="NewHuntWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="newHuntButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>New Hunt!</string>
   </property>
  </widget>
  <widget class="QLabel" name="pokemonLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pokemon:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game:</string>
   </property>
  </widget>
  <widget class="QLabel" name="methodLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Method:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pokemonComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="gameComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="methodComboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="methodLabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Odds:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="oddsSpinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>8192</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
